presto-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/docker/files/apply-site-xml-override.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
`apply-site-xml-override.sh` should be moved, it isn't used by anyone else, is it?

(same for `site-override.xslt`)

Reply:
Where should I place it?

Reply:
for now -- in envornment-specific directory; then you can use it directly from there, without additional binds

Reply:
later we should move this to docker-images